src/olympia/devhub/templates/devhub/new-landing/components/overview.html

```diff
@@ -2,12 +2,12 @@
   <div>
     <h1>{{ _('Customize Your Firefox') }}</h1>
     <p>
-      {% trans %}
+      {% trans html_link="https://developer.mozilla.org/docs/Web/HTML", js_link="https://developer.mozilla.org/docs/Web/JavaScript", css_link="https://developer.mozilla.org/docs/Web/CSS" %}
       Add-ons let millions of Firefox users enhance their browsing experience.
-      If you know <a href="#">HTML</a>, <a href="#">JavaScript</a>, and <a href="#">CSS</a>,
+      If you know <a href="{{ html_link }}">HTML</a>, <a href="{{ js_link }}">JavaScript</a>, and <a href="{{ css_link }}">CSS</a>,
       you already have all the necessary skills to make a great add-on.
       {% endtrans %}
     </p>
-    <a href="#" class="Button Button--primary">{{ _('Learn How to make an Add-on') }}</a>
+    <a href="https://developer.mozilla.org/en-US/Add-ons" class="Button Button--primary">{{ _('Learn How to make an Add-on') }}</a>
```

Comment:
Fixed in 81356c4811, pretty much them are translated so I removed `en-US` for all of the links. We default to `en-US` anyway if there's no translation.